CodeConverter/Shared/DefaultReferences.cs

```diff
@@ -29,8 +29,7 @@ public static class DefaultReferences
             typeof(System.Web.HttpUtility),
             typeof(System.Xml.XmlElement),
             typeof(System.Xml.Linq.XElement),
-            typeof(Microsoft.VisualBasic.Constants),
-            typeof(System.Data.SqlClient.SqlCommand),
+            typeof(Microsoft.VisualBasic.Constants)
```

Comment:
ANy reason to remove SqlCommand?

Reply:
Good catch. I initially thought it was unnecessary because of system.data.datarow but I see sqlcommand has been moved to its own assembly in later frameworks.

Reply:
Good catch. I initially thought it wasn't necessary with datarow just above, but I see that sqlcommand has been moved to its own assembly in other framework versions. 

> Looks good, just the one thing to check commented above.
> 
> Thanks for the PRs, they look great. I'm also all for using newer, terser syntax. But could I ask that in future PRs you try to separate formatting/tidying changes from behaviour changes?

Yes good call and sorry about that, that's easier to review.